.github/workflows/semgrep.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - uses: actions/checkout@v3
       - run: semgrep ci
         env:
-           SEMGREP_RULES: p/default
+           SEMGREP_RULES: r/all
```

Comment:
like https://registry.semgrep.dev/ruleset? Is it possible to add small incremental set?
Enabling all rules will be over verbose.